### PR TITLE
Use modern php versions

### PR DESCRIPTION
**Pull request by vkhramtsov**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |
| License       | MIT

### CODE REVIEW

**Comment by vkhramtsov** [comment]
@sstok I would like to update minimum requirement for symfony 4 to symfony 4.1 due to https://symfony.com/roadmap/4.1 here and here https://github.com/rollerworks/PasswordStrengthValidator. What do you think about it?

**Comment by vkhramtsov** [comment]
@sstok Changed configuration of phpstan a little bit. Looks like now this PR is ready for review

File: phpstan.neon [hunk 1]
```diff
@@ -1,8 +1,7 @@
 parameters:
     autoload_files:
         - vendor/autoload.php
-        - vendor/symfony/phpunit-bridge/bin/.phpunit/phpunit-5.7.19/vendor/autoload.php
+        - vendor/bin/.phpunit/phpunit-7.5.6/vendor/autoload.php
 
     ignoreErrors:
-        - '#Class Symfony\\Component\\ExpressionLanguage\\ExpressionLanguage not found and could not be autoloaded#'
-        - '#Class Symfony\\Component\\Config\\Definition\\Builder\\TreeBuilder does not have a constructor and must be instantiated without any parameters#'
+        - '#Call to an undefined method Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition::addDefaultsIfNotSet().#'
```
Comment (sstok) [inline phpstan.neon]:
```suggestion
        - '#Call to an undefined method Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition\:\:addDefaultsIfNotSet\(\)#'
```

Comment (vkhramtsov) [reply phpstan.neon]: Fixed

**Comment by sstok** [comment]
Thank you @vkhramtsov

**Comment by vkhramtsov** [comment]
@sstok No problem. Looks like #95 could be closed as it duplicates this PR

**Pull request merged**